be/src/exec/vectorized/olap_scan_node.cpp

```diff
@@ -173,7 +173,7 @@ Status OlapScanNode::close(RuntimeState* state) {
     if (is_closed()) {
         return Status::OK();
     }
-    RETURN_IF_ERROR(exec_debug_action(TExecNodePhase::CLOSE));
+    exec_debug_action(TExecNodePhase::CLOSE);
```

Comment:
could we directly remove exec_debug_action method call?

Reply:
`exec_debug_action` will simulate the delayed condition. It will sleep 1s instead of returning error if `_action==WAIT`.